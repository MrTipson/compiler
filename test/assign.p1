void putint(int val);
int write(int fd, char* buf, int count);

void main() {
	char ln = '\n';
	char c = '5';
	c -= '0';
	putint((int)c);
	write(1,&ln,1);
	int a = 9;
	a += '0';
	write(1,(char*)&a,1);
	write(1,&ln,1);
	int b = 160;
	b /= 10;
	putint(b);
	write(1,&ln,1);
	b *= 2;
	putint(b);
	write(1,&ln,1);
}