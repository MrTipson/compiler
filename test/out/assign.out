5
9
16
32
